src/common.c

```diff
@@ -124,6 +124,31 @@ const char *dwg_bits_name[] =
   "3DD",
   "CRC",
   "CRC64",
+  "SPLINE_control_point",
```

Comment:
Do you want create another one array with `SPLINE_` like structure's minimal sizes and use it in REPEAT macros?
Or any other ideas?

Reply:
We might need a seperate size array for all subtypes. 
I also need more for `work/dynapi`, like subtype of, size of vars, offset, but this will be seperate.

Reply:
Should I remove first commit (with composite type's sizes) form this pull request?